contracts/Portfolio.sol

```diff
@@ -806,13 +811,13 @@ abstract contract PortfolioVirtual is Objective {
         (, bytes1 instruction) = AssemblyLib.separate(data[0]); // Upper byte is useMax, lower byte is instruction.
 
         if (instruction == FVM.ALLOCATE) {
-            (uint8 useMax, uint64 poolId, uint128 deltaLiquidity) =
+            (uint8 useMax, uint64 poolId, uint128 deltaLiquidity, uint128 deltaAsset, uint128 deltaQuote) =
```

Comment:
personally, I find the `deltaAsset` and `deltaQuote` names misleading in this context. It sounds like these are the ones that the protocol will use, but these are the min/max amounts. Could consider something like `desiredDeltaAsset` or `expectedDeltaAsset` instead.

Reply:
That's reasonable, I'll change these names.